elastic/kibana
# [Fleet] Fix add agent authentication settings display when no tokens

## Summary

Resolve #120867

When there is no enrollment token we should the callout to generate a new enrollment token (as it's done on the agent policy page) on the agent page.

That PR address that 

## UI Changes

<img width="1531" alt="Screen Shot 2021-12-15 at 3 00 09 PM" src="https://user-images.githubusercontent.com/1336873/146268602-c7ba1ba5-fd72-46f3-93d0-efc831130943.png">


## How to test

From an environment with a fleet server setup
* go to the enrollment token page and revoke the enrollment token for the default policy
* go to the agent page and try to add a agent, you should see the no enrollment token callout
